helm/vitess/templates/_vttablet.tpl

```diff
@@ -176,9 +176,10 @@ volumes:
 {{- $tablet := index . 4 -}}
 {{- with $tablet.vttablet -}}
 {{- $0 := $.Values.vttablet -}}
+{{- $cellClean := $cell.name | replace "_" "-" -}}
```

Comment:
As discussed offline, please fold this into the "format-shard-name" function and rename it to e.g. "clean-label-name".